python-backend/app/api/parties/party_appt/models/mine_party_appt.py

```diff
@@ -104,6 +104,13 @@ def find_by_party_guid(cls, _id):
         except ValueError:
             return None
 
+    @classmethod
+    def find_by_permit_guid(cls, _id):
+        try:
+            return cls.find_by(permit_guid=_id)
+        except ValueError:
```

Comment:
This try catch for the value error can be remove. The global handler will catch this.

Reply:
Removed